infra/cifuzz/cifuzz.py

```diff
@@ -187,8 +192,18 @@ def run_fuzzers(fuzz_seconds, workspace):
 
   # Run fuzzers for alotted time.
   for fuzzer_path in fuzzer_paths:
-    target = fuzz_target.FuzzTarget(fuzzer_path, fuzz_seconds_per_target,
-                                    out_dir)
+
+    # OSS-Fuzz specific project setup.
+    if project_name:
+      corpus_dir = download_latest_corpus(project_name, out_dir,
+                                          os.path.basename(fuzzer_path))
+      target = fuzz_target.FuzzTarget(fuzzer_path,
+                                      fuzz_seconds_per_target,
+                                      out_dir,
+                                      corpus_dir=corpus_dir)
+    else:
```

Comment:
you can reduce the amount of code here if you define `corpus_dir = None` on line 196, then leave lines 197-199 as is, and then have a single `FuzzTarget` call

Reply:
Fixed.